pkg/ledger/resolver.go

```diff
@@ -18,13 +18,6 @@ func (fn ResolveOptionFn) apply(r *Resolver) error {
 	return fn(r)
 }
 
-func WithStorageDriver(factory storage.Driver) ResolveOptionFn {
```

Comment:
For an unknown reason, the WithStorageDriver function is called with a *sqlstorage.Driver object instead of the expected *opentelemetrytraces.openTelemetryStorageDriver.

Nevermind, this function is useless since the NewResolver function already take the storage.Driver interface as first parameter (which have the good type...)